public/state-of-charge/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="nl">
+  <head>
+    <title>Chargetrip API: state of chargeexample</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="description" content="Build a route with the Chargetrip API" />
```

Comment:
Typo + description can be clearer.

Reply:
I agree. I change it to: Chargetrip API: example of the impact of your SOC
But I am not sure if that is much better, do you have suggestions?